pkg/server/singleprocess/service_runner.go

```diff
@@ -881,6 +888,10 @@ func (s *Service) handleJobStreamRequest(
 	log.Trace("event received", "event", req.Event)
 	switch event := req.Event.(type) {
 	case *pb.RunnerJobStreamRequest_Complete_:
+		if event.Complete == nil {
+			log.Error("event.Complete was not filled when we expected it to be", "id", job.id)
+			return hcerr.Externalize(log, err, "failed to complete job", "id", job.id)
```

Comment:
I think the linter is mad about this one? There's no err available on this branch I think.

Reply:
Fixed, I think my vim plugin stopped working and didn't catch that `job.id` should actually be `job.Id` 😨